<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKOutputSave2FileDialog</class>
 <widget class="QDialog" name="SAKOutputSave2FileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters Setting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>File Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="pathLineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="selectPushButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Outputing file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="okPushButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="truncatePushButton">
     <property name="toolTip">
      <string>All earlier data will be clean</string>
     </property>
     <property name="text">
      <string>Truncate</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Parameters Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="1">
       <widget class="QRadioButton" name="utf8RadioButton">
        <property name="text">
         <string>Utf8</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="timestampCheckBox">
        <property name="text">
         <string>Save timestamp</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="readDataCheckBox">
        <property name="text">
         <string>Save read data</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="writtenDataCheckBox">
        <property name="text">
         <string>Save written data</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="binRadioButton">
        <property name="text">
         <string>Bin</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="hexRadioButton">
        <property name="text">
         <string>Hex</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel{
    color:green
}</string>
        </property>
        <property name="text">
         <string>Text format</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">QLabel{
    color:green
}</string>
        </property>
        <property name="text">
         <string>Saving options</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SAKOutputSave2FileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
